---
title: TECtool
summary: TECtool (Terminal Exon Characterization tool) is a Machine Learning-based annotation software (implemented in Python) that makes use of RNA sequencing (RNA-seq) data to identify novel terminal exons, infer novel transcript isoforms and annotate potential coding sequences.

#authors:
# - stefan_dentro
# - david_wedge
# - jdemeul
# - nansari_pour

#date: "2012-05-25"
lastmod: "2018-10-01"
external_link: 
image:
  caption: 
  focal_point: Smart
links:
- name: Website
  url: http://tectool.unibas.ch
- name: Publication
  url: https://www.nature.com/articles/s41592-018-0114-z
tags:
- "MainCorrespondingAuthorship"
- "SoftwareTools"
- "MachineLearning"
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

---

Sequencing of RNA 3' ends has uncovered numerous sites that do not correspond to the termination sites of known transcripts. Through their 3' untranslated regions, protein-coding RNAs interact with RNA-binding proteins and microRNAs, which regulate many properties, including RNA stability and subcellular localization. We developed the terminal exon characterization (TEC) tool (http://tectool.unibas.ch ), which can be used with RNA-sequencing data from any species for which a genome annotation that includes sites of RNA cleavage and polyadenylation is available. We discovered hundreds of previously unknown isoforms and cell-type-specific terminal exons in human cells. Ribosome profiling data revealed that many of these isoforms were translated. By applying TECtool to single-cell sequencing data, we found that the newly identified isoforms were expressed in subpopulations of cells. Thus, TECtool enables the identification of previously unknown isoforms in well-studied cell systems and in rare cell types.

