---
title: KAPAC
summary: The KAPAC tool enables the identification of sequence motifs that can explain changes in cellular 3' end processing from high-throughput sequencing data.

#authors:
# - stefan_dentro
# - david_wedge
# - jdemeul
# - nansari_pour

#date: "2012-05-25"
lastmod: "2018-03-28"
external_link: 
image:
  caption: 
  focal_point: Smart
links:
- name: Website
  url: https://github.com/zavolanlab/PAQR_KAPAC
- name: Publication
  url: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1415-3
tags:
- "MainAuthorship"
- "SoftwareTools"
- "MachineLearning"
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

---

3' Untranslated regions (3' UTRs) length is regulated in relation to cellular state. To uncover key regulators of poly(A) site use in specific conditions, we have developed KAPAC, standing for k-mer activity on polyadenylation site choice. KAPAC infers activities of oligomeric sequence motifs on poly(A) site choice. Previously, we have applied KAPAC to sequencing data from normal and tumor tissue samples and uncovered motifs that can explain changes in cleavage and polyadenylation in specific cancers. Amongst others, our analysis pointed to the polypyrimidine tract binding protein 1 (PTBP1) as a regulator of poly(A) site choice in glioblastoma [Gruber et al. 2018]({{< ref "/publication/2018_KAPAC_PAQR_GenomeBiology/index.md" >}}).

