---
title: KAPAC
summary: The tool enables the identification of sequence motifs that can explain changes in cellular 3' end processing from high-throughput sequencing data. KAPAC (implemented in R) enables the inference of regulatory sequence motifs that shape the usage of 3' end processing sites under specific cellular conditions.

#authors:
# - stefan_dentro
# - david_wedge
# - jdemeul
# - nansari_pour

#date: "2012-05-25"
lastmod: "2018-03-28"
external_link: 
image:
  caption: 
  focal_point: Smart
links:
- name: Website
  url: https://github.com/zavolanlab/PAQR_KAPAC
- name: Publication
  url: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1415-3
tags:
- "MainCorrespondingAuthorship"
- "SoftwareTools"
- "MachineLearning"
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

---

KAPAC, standing for k-mer activity on polyadenylation site choice, infers activities of oligomeric sequence motifs on poly(A) site choice.

