---
title: Human | Data | Society - Personalized medicine 

##event: Wowchemy Conference
##event_url: https://example.org

##location: Wowchemy HQ
##address:
##  street: 450 Serra Mall
##  city: Stanford
##  region: CA
##  postcode: '94305'
##  country: United States

##summary: An example talk using Wowchemy's Markdown slides feature.
##abstract: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellusac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum. Sed ac faucibus dolor, scelerisque sollicitudin nisi. Cras purus urna, suscipit quis sapien eu, pulvinar tempor diam."

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: "2023-01-08T11:00:00Z"
date_end: "2023-01-08T11:00:00Z"
##all_day: false

# Schedule page publish date (NOT talk date).
publishDate: "2023-01-08T11:00:00Z"

authors: []
tags: []

# Is this a featured talk? (true/false)
featured: false

image:
  placement: 1
  focal_point: "Center"
#  alt_text: APA
#  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/bzdhc5b3Bxs)'

#links:
#- icon: twitter
#  icon_pack: fab
#  name: Follow
#  url: https://twitter.com/georgecushen
#url_code: ""
#url_pdf: ""
#url_slides: ""
#url_video: ""

# Markdown Slides (optional).
#   Associate this talk with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
##slides: example

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
##projects:
## - projects = ["genomics-england"]
##projects = ["genomics-england"]
---

## Unraveling molecular signatures for personalized health care

Currently available therapies are mostly designed for the "average" patient and neglect patient-to-patient heterogeneity. However, these 'one-size-fits-all' treatment approaches are not suitable for most complex diseases, such as cancers, as they do not take into account the molecular properties of an individual disease. In our research we develop data science tools and approaches that enable to extract previously unknown molecular signatures and unlock their biomarker potential for personalized medicine. By analyzing the signatures in terms of their occurrence within certain cancer types and their association with disease progression, we develop predictive models that can support personalized diagnostics and clinical decision making in the future. Personalized medicine can improve the lives of patients and clinical professionals lastingly. This is one of the ultimate aims of our research group, which is also a member of the recently established [**Centre for Human Data Society**](https://www.uni-konstanz.de/forschen/forschungseinrichtungen/centre-for-human-data-society/).

## Related Publications

 - **Gruber AJ**, Zavolan M, *“Alternative cleavage and polyadenylation in health and disease.”*, ***Nature Reviews Genetics***. 2019 Oct;20(10):599-614. doi: 10.1038/s41576-019-0145-z. [[*Publication*]](https://www.nature.com/articles/s41576-019-0145-z)
 - Balwierz PJ, Pachkov M, Arnold P, **Gruber AJ**, Zavolan M, and van Nimwegen E, *“ISMARA: automated modeling of genomic signals as a democracy of regulatory motifs.”*, ***Genome Research*** 2014 May;24(5):869-84. doi: 10.1101/gr.169508.113. [[*Publication*]](https://genome.cshlp.org/content/24/5/869.long) [[*Data Science Tool*]](https://ismara.unibas.ch/mara/)
 - Islam SMA, Díaz-Gay M, Wu Y, Barnes M, Vangara R, Bergstrom EN, He Y, Vella M, Wang J, Teague JW, Clapham P, Moody S, Senkin S, Li YR, Riva L, Zhang T, **Gruber AJ**, Steele CD, Otlu B, Khandekar A, Abbasi A, Humphreys L, Syulyukina N, Brady SW, Alexandrov BS, Pillay N, Zhang J, Adams DJ, Martincorena I, Wedge DC, Landi MT, Brennan P, Stratton MR, Rozen SG, Alexandrov LB, *“Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor.”*, ***Cell Genomics***. 2022 Nov 9;2(11). doi: 10.1016/j.xgen.2022.100179. [[*Publication*]](https://www.sciencedirect.com/science/article/pii/S2666979X22001240?via%3Dihub) [[*Data Science Tool*]](https://github.com/AlexandrovLab/SigProfilerExtractor)
 - **Gruber AJ**, Schmidt R, Ghosh S, Martin G, Gruber AR, van Nimwegen E, Zavolan M, *“Discovery of physiological and cancer-related regulators of 3' UTR processing with KAPAC.”*, ***Genome Biology***, 2018 Mar 28;19(1):44. doi: 10.1186/s13059-018-1415-3 [[*Publication*]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1415-3) [[*Data Science Tool*]](https://github.com/zavolanlab/PAQR_KAPAC)
 - Ferrarini MG, Lal A, Rebollo R, **Gruber AJ**, Guarracino A, Gonzalez IM, Floyd T, de Oliveira DS, Shanklin J, Beausoleil E, Pusa T, Pickett BE, Aguiar-Pulido V., *“Genome-wide bioinformatic analyses predict key host and viral factors in SARS-CoV-2 pathogenesis.”*, ***Communications Biology***, 2021 May 17;4(1):590. doi: 10.1038/s42003-021-02095-0. [[*URL*]](https://www.nature.com/articles/s42003-021-02095-0)
 - Lal A, Galvao Ferrarini M, **Gruber AJ**, *“Investigating the Human Host-ssRNA Virus Interaction Landscape Using the SMEAGOL Toolbox.”*, ***Viruses***. 2022 Jun 29;14(7):1436. doi: 10.3390/v14071436. [[*Publication*]](https://www.mdpi.com/1999-4915/14/7/1436) [[*Data Science Tool*]](https://github.com/gruber-sciencelab/SMEAGOL)
 - Hoelzer M, Krähling V, Amman F, Barth E, Bernhart SH, Carmelo VA, Collatz M, Doose G, Eggenhofer F, Ewald J, Fallmann J, Feldhahn LM, Fricke M, Gebauer J, **Gruber AJ**, Hufsky F, Indrischek H, Kanton S, Linde J, Mostajo N, Ochsenreiter R, Riege K, Rivarola-Duarte L, Sahyoun AH, Saunders SJ, Seemann SE, Tanzer A, Vogel B, Wehner S, Wolfinger MT, Backofen R, Gorodkin J, Grosse I, Hofacker I, Hoffmann S, Kaleta C, Stadler PF, Becker S, Marz M, *"Differential transcriptional responses to Ebola and Marburg virus infection in bat and human cells."*, ***Scientific Reports***, 2016 Oct 7;6:34589. doi: 10.1038/srep34589. [[*URL*]](https://www.nature.com/articles/srep34589)



