---
title: "A comprehensive analysis of 3′ end sequencing data sets reveals novel polyadenylation signals and the repressive role of heterogenous ribonucleoprotein C on cleavage and polyadenylation"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- Andreas J. Gruber
- Ralf Schmidt
- Andreas R. Gruber
- Georges Martin
- Souvik Ghosh
- Manuel Belmadani
- Walter Keller
- Mihaela Zavolan

# Author notes (optional)
#author_notes:
#- "Corresponding author & equal contribution"
#- "Corresponding author & equal contribution"

date: "2016-08-26T00:00:00Z"
doi: "10.1101/gr.202432.115"

# Schedule page publish date (NOT publication's date).
publishDate: "2016-08-26T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: Genome Research
publication_short: Genome Res.

abstract: Alternative polyadenylation (APA) is a general mechanism of transcript diversification in mammals, which has been recently linked to proliferative states and cancer. Different 3’ untranslated region (3’ UTR) isoforms interact with different RNA-binding proteins (RBPs), which modify the stability, translation, and subcellular localization of the corresponding transcripts. Although the heterogeneity of pre-mRNA 3’ end processing has been established with high-throughput approaches, the mechanisms that underlie systematic changes in 3’ UTR lengths remain to be characterized. Through a uniform analysis of a large number of 3’ end sequencing data sets we have uncovered 18 signals, 6 of which novel, whose positioning with respect to pre-mRNA cleavage sites indicates a role in pre-mRNA 3’ end processing in both mouse and human. With 3’ end sequencing we have demonstrated that the heterogeneous ribonucleoprotein C (HNRNPC), which binds the poly(U) motif whose frequency also peaks in the vicinity of polyadenylation (poly(A)) sites, has a genome-wide effect on poly(A) site usage. HNRNPC-regulated 3’ UTRs are enriched in ELAV-like RNA binding protein 1 (ELAVL1) binding sites and include those of the CD47 gene, which participate in the recently discovered mechanism of 3’ UTR-dependent protein localization (UDPL). Our study thus establishes an up-to-date, high-confidence catalog of 3’ end processing sites and poly(A) signals and it uncovers an important role of HNRNPC in regulating 3’ end processing. It further suggests that U-rich elements mediate interactions with multiple RBPs that regulate different stages in a transcript’s life cycle.

# Summary. An optional shortened abstract.
summary: 

tags: []
# - main

# Display this page in the Featured widget?
featured: true

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
##projects:
##- example

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
##slides: example
---

